Configurations/50-cppbuilder.conf

```diff
@@ -58,5 +58,64 @@ my %targets = (
         shared_defflag   => '',
         perl_platform    => 'Windows::cppbuilder',
         uplink_arch      => 'common',
+    },
+    "BC-64" => {
+        inherit_from     => [ "BASE_Windows" ],
+        sys_id           => "WIN64",
+        bn_ops           => "BN_LLONG",
+        thread_scheme    => "winthreads",
+        cc               => "bcc64",
+        CPP              => "cpp64 -oCON -Sc -Sr",
+        defines          => add("WIN64_LEAN_AND_MEAN", "OPENSSL_SYS_WIN64",
+                                "L_ENDIAN", "DSO_WIN64", "_stricmp=stricmp",
```

Comment:
What is `DSO_WIN64`? I can find `DSO_WIN32` used in OpenSSL code, but not this one

Reply:
fix it here https://github.com/openssl/openssl/pull/23377/commits/25d2277742cfb64c1eed85dbcb0a9f684d94404b